master/pkg/tasks/task_gc.go

```diff
@@ -28,6 +28,7 @@ type GCCkptSpec struct {
 	// and just refresh the state of the checkpoint.
 	CheckpointGlobs    []string
 	DeleteTensorboards bool
+	TensorboardOnly    bool
```

Comment:
looking at `gc_checkpoints.py`, i think we should be able to write an empty "ToDelete" and pass `DeleteTensorboards: true` and have everything deleted. but probably i am missing something, so why did we add this `TensorboardOnly` bool?

Reply:
I was using `TensorboardOnly` to create a unique call to `gc_chekcpoints.py` that doesn't have any flags related to checkpoints. This did not work, I have updated to code to run checkpointGC with  `DeleteTensorboards: true` and an empty `ToDelete`. Unfortunately this is also not working. Currently trying to debug

Reply:
Got it working by modifying `task_gc.go`